### PR TITLE
<TimePicker />

### PR DESCRIPTION
**Pull request by ilyaagarkov**
add time-picker based on https://antonfisher.com/react-simple-timefield/

### CODE REVIEW

File: src/time_picker/index.es [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <TimePickerFormFiled
+        ref={this.setEl('input')}
+        name={name}
+        label={label}
+        error={error}
+        required={required}
+        disabled={disabled}
+        onChange={this.actionChange(onChange)}
+        className={className}
+
```
Comment (atomiomi) [inline src/time_picker/index.es]: pass value. get from state

File: src/time_picker/time-picker-input.es [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      onChange={onChange}
+      style={{ width: 'iv' }} // iv - invalid value hack for reset react-simple-timefield style
+    />
+  ),
+);
+
+TimePickerInput.propTypes = {
+  className: propTypes.string,
+  name: propTypes.string,
+  value: propTypes.func,
```
Comment (atomiomi) [inline src/time_picker/time-picker-input.es]: fix prop type

File: src/time_picker/time-picker-input.es [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  ({ className, onChange, value, name, required, disabled }, ref) => (
+    <TimeField
+      ref={ref}
+      value={value}
+      name={name}
+      required={required}
+      disabled={disabled}
+      className={className}
+      onChange={onChange}
+      style={{ width: 'iv' }} // iv - invalid value hack for reset react-simple-timefield style
```
Comment (atomiomi) [inline src/time_picker/time-picker-input.es]: width: undefined

File: preview/containers/form/index.es [hunk 4]
```diff
@@ -162,6 +163,10 @@ class FormPreview extends PureComponent {
               />
             </FormGroup>
 
+            <FormGroup>
+              <TimePicker label="Time Picker" error="Required" required />
```
Comment (atomiomi) [inline preview/containers/form/index.es]: add to forms

File: src/time_picker/index.es [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import InputComponent from '../base/index';
+import { TimePickerFormFiled } from './time-picker-form-filed';
+
+export default class TimePicker extends InputComponent {
```
Comment (atomiomi) [inline src/time_picker/index.es]: 2 empty lines after imports

File: src/time_picker/time-picker-form-filed.es [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import cn from 'clsx';
+import { TimePickerInput } from './time-picker-input';
+
+export const TimePickerFormFiled = React.forwardRef(
```
Comment (atomiomi) [inline src/time_picker/time-picker-form-filed.es]: 2 empty lines after import

File: package.json [hunk 7]
```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-form",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-form/issues",
-  "version": "8.2.5",
+  "version": "8.3.0-beta.3",
```
Comment (suprMax) [inline package.json]: publish as stable? LGTM

File: src/time_picker/index.es [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    const input = <input type="text" ref={this.setEl('input')} />;
+
+    return (
+      <label className={className}>
+        {labelNode}
+        <div className="c-input-container">
+          <TimeField
+            {...cleanProps}
+            className={inputClassName}
+            style={style}
```
Comment (suprMax) [inline src/time_picker/index.es]: could use shorthand syntax for style->value

**Pull request closed without merging**